### PR TITLE
Webpack Dev Server

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    'prop-types': 'propTypes',
+  },
+  resolve: {
+    extensions: ['.js', '.jsx'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /(node_modules|bower_components)/,
```
Comment (lalli-flores) [inline webpack.config.js]: We don't use Bower
```suggestion
        exclude: /(node_modules)/,
```

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    open: true,
+    overlay: true,
+    hot: true,
+    port: 1234,
+  },
+  plugins: [
+    new CleanWebpackPlugin({}),
+    new HtmlWebpackPlugin({
+      filename: 'index.html',
+      template: 'src/index.html',
```
Comment (lalli-flores) [inline webpack.config.js]: As discussed in the Teams messages, we should probably have this be on a `demo` folder outside `src`

File: webpack.config.js (hunk 1 above)
Comment (adamweeks) [inline webpack.config.js]: You can remove `bower_components`

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
+const path = require('path');
+const webpack = require('webpack');
+const {CleanWebpackPlugin} = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  cache: true,
+  node: {
```
Comment (lalli-flores) [inline webpack.config.js]: Why are you overwriting this?

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        ],
+      },
+    ],
+  },
+  devServer: {
+    contentBase: path.resolve(__dirname, './src'),
+    open: true,
+    overlay: true,
+    hot: true,
+    port: 1234,
```
Comment (adamweeks) [inline webpack.config.js]: could you add `https: true,` here? Browsers are starting to get very strict on https requirement for local media access.

**Comment by akoushke** [comment]
I can't push to this PR anymore. will create another PR and addressed the comment there.

**Comment by lalli-flores** [comment]
@akoushke next time ask for help. We lose the history otherwise.
Replaced by https://github.com/webex/widgets/pull/7

**Pull request closed without merging**